experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -387,6 +466,19 @@ export function makeOpGenerator(
 	};
 }
 
+function checkContainerConnections(containers: readonly ContainerInfo[] | undefined) {
+	if (containers === undefined) {
+		return false;
+	}
+	for (const containerInfo of containers) {
+		const connection = containerInfo.container.connectionState;
+		if (connection === 2 || connection === 0) {
```

Comment:
Can we use `ConnectionState` from `container-definitions` instead of hardcoding the numerical values? Would make this more readable.

Reply:
updated to use `ConnectionState` enum